zebra/zebra_dplane.c

```diff
@@ -210,6 +210,7 @@ struct dplane_ctx_rule {
 	uint8_t dsfield;
 	struct prefix src_ip;
 	struct prefix dst_ip;
+	char ifname[INTERFACE_NAMSIZ + 1];
```

Comment:
it looked like the code had been using the existing 'zd_ifname' field - was that not working somehow? (it's still being initialized, isn't it?)

Reply:
yep it's the same thing and was `wrong` to have it there in the first place.  Future commits from me will reduce the duplication.  Left it alone right now since I'm focussing on bug fixes.

Reply:
interesting - why was that wrong? I've sometimes reused existing fields myself, I think - since their only available through accessor apis, it sort of seemed ok to do that?

Reply:
the ifindex/interface name is a property of the rule itself.  We were storing the ifindex in the `struct pbr_rule` but the ifname in the `struct zebra_pbr_rule`.  So we had duplication of data and it's part of the rule not part of the data structure that we are using to track the rules. imo.